### DISCUSSION TITLE
支持windows7

### DISCUSSION THREAD

**Discussion by beicause** [v2rayA/v2rayA#1463] [General]
自从 2d2c3a53103362fd40929e5d3904da896981a7fa，v2raya的windows安装包所需的最小版本为windows10。此前的2.2.5.2版本，v2raya在windows7安装时也会报错。希望能支持windows7的老电脑。

![微信图片_20240801092343](https://github.com/user-attachments/assets/c94ff842-39bd-4581-a8b8-c1f8005c0687)

**Comment by eronez** [comment]
1. Stay on the last v2raya version that can be used on Windows 7.
2. Switch to another software (clash verge or v2rayN)
